ci/Dockerfile.test

```diff
@@ -11,7 +11,7 @@ COPY ci/application.conf /opt/spark-jobserver/bin/application.conf
 
 RUN sbt job-server-extras/assembly job-server-tests/package buildPython buildPyExamples
 
-RUN cp /opt/spark-jobserver/job-server-extras/target/scala-2.10/spark-job-server.jar /opt/spark-jobserver/bin/spark-job-server.jar && \
+RUN cp /opt/spark-jobserver/job-server-extras/target/scala-2.11/spark-job-server.jar /opt/spark-jobserver/bin/spark-job-server.jar && \
```

Comment:
Can you revert the change in this one?

Reply:
Also let add checkout 0.7.0 of spark-job-server and see how it works

after git clone may be 

RUN git checkout 8e2f459

Reply:
I create branch named release/0.7.0